website/integrations/documentation/glpi/index.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Integrate with GLPI (SAML)
+sidebar_label: GLPI
+support_level: community
+---
+
+## What is GLPI
+
+> GLPI (Gestionnaire Libre de Parc Informatique) is an open-source IT asset management and service desk software. It helps organizations manage hardware, software, tickets, users, and IT services in a centralized platform.  
+> https://www.glpi-project.org
+
+## Preparation
+
+:::info
+This documentation lists only the settings that you need to change from their default values. Be aware that any changes other than those explicitly mentioned in this guide could cause issues accessing your application.
+:::
+
+By default GLPI only offers OAuth that is only available to subscribers. This guide is for a community plugin named `samlsso` by DonutsNL. (GLPI v11+)
+
+> https://github.com/DonutsNL/samlsso
+
+1. Download latest release `samlsso.zip` and unpack it in `glpiroot/data/marketplace`
+2. Activate the plugin in **Setup** > **Plugins**
+
+## GLPI configuration base
+
+1. Add a new instance in **Setup** > **samlSSO**
+
+- **General**
+    - Friendly name: authentik
+    - Login icon: find an icon from [FontAwesome](https://fontawesome.com/)
+    - Is active: true
+- **Transit**
+    - Validate XML: true
+- **Security**
+    - Strict: true
+    - Jit user creation: true
+
+2. Click **Save** to create an instance
+
+## authentik configuration
+
+To support the integration of GLPI with authentik, you need to create an application/provider pair in authentik.
+
+### Create an application and provider in authentik
+
+1. Log in to authentik as an administrator and open the authentik Admin interface.
+2. Navigate to **Applications** > **Applications** and click **Create with Provider** to create an application and provider pair. (Alternatively you can first create a provider separately, then create the application and connect it with the provider.)
+
+- **Application**: provide a descriptive name, an optional group for the type of application, the policy engine mode, and optional UI settings.
+- **Choose a Provider type**: select **SAML Provider** as the provider type.
+- **Configure the Provider**: provide a name (or accept the auto-provided name), the authorization flow to use for this provider, and the following required configurations.
+    - ACS URL: `created GLPI samlSSO instance > Service Provider > AcsUrl`
```

Comment:
To better match the existing integrations, instead of putting where you find the acsurl, can you put in a placeholder?

for example,
`https://glpi.company/saml/acs`